scripts/remove_stale_ci_builds.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
probably should delete this file for the time being (apologies for adding it)

Reply:
Yep